docs/opentracing.md

```diff
@@ -57,8 +57,9 @@ https://www.jaegertracing.io/docs/latest/getting-started.
 ## Enable OpenTracing in Synapse
 
 OpenTracing is not enabled by default. It must be enabled in the
-homeserver config by uncommenting the config options under `opentracing`
-as shown in the [sample config](./sample_config.yaml). For example:
+homeserver config by adding the `opentracing` option to your config file. You can find 
+documentation about how to do this in the [config manual](usage/configuration/config_documentation.md) 
+under the header `opentracing`. See below for an example opentracing configuration: 
```

Comment:
```suggestion
documentation about how to do this in the [config manual under the header 'Opentracing'](usage/configuration/config_documentation.md#opentracing).
See below for an example Opentracing configuration: 
```
You can link straight to the header (either by guessing, or by visiting the docs and seeing what the `#anchor-name` turns out to be).

We don't seem to consistently capitalise the `T` all the time but at least the `O` should be I think.